src/aspire/source/coordinates.py

```diff
@@ -44,7 +45,7 @@ class CoordinateSource(ImageSource, ABC):
     This also allows the CoordinateSource to be saved to an `.mrcs` stack.
     """
 
-    def __init__(self, files, particle_size, max_rows):
+    def __init__(self, files, particle_size, max_rows, B):
```

Comment:
Eh do we really want `B` to be a required parameter? It's also not clear to me that we want to have `B` at all.